src/patternfly/components/AppLauncher/app-launcher-toggle.hbs

```diff
@@ -0,0 +1,5 @@
+<button id="{{id}}-button" class="pf-c-app-launcher__toggle{{#if app-launcher-toggle--modifier}} {{app-launcher-toggle--modifier}}{{/if}}"
```

Comment:
I just noticed that here and the kebab dropdown that the dropdown is only 28px tall. Should that match the other dropdowns? Would it be crazy to replace this and the kebab dropdowns with `.pf-c-button.pf-m-plain`?